app/views/pages/supporters.html.erb

```diff
@@ -0,0 +1,42 @@
+<% @title = t('.title') %>
+
+<%= prose do %>
+  <p>
+    RubyGems.org is a critical part of the Ruby ecosystem, processing millions of downloads each day and enabling developers around the world to build with confidence. As a community-run service overseen by Ruby Central, a 501(c)(3) nonprofit, we rely on the support of individuals and organizations who believe in open source and want to keep RubyGems.org secure and reliable long-term.  Specifically, we fund this service through generous support from RubyGems Supporters, Ruby Central’s Corporate Sponsorship Program, Ruby Central memberships, and Infrastructure Sponsors.
+  </p>
+  <p>
+    The RubyGems Supporter Program is an easy way for you or your company to contribute to the maintenance, operations, and growth of the open source infrastructure you depend on. You can find out more about this program <a href="https://rubycentral.org/open-source/">here</a>.
+  </p>
+
+  <h2>Want to support RubyGems.org?</h2>
+  <p>
+    <a href="https://rubycentral.org/open-source/#/portal/signup">Become a Supporter</a>
```

Comment:
What are your thoughts about making the this call-to-action more prominent? Like a distinctive button or something?